keel-web/src/main/kotlin/com/netflix/spinnaker/keel/controllers/v1/admin/PolicyController.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.keel.controllers.v1.admin
+
+import com.netflix.spinnaker.keel.Policy
+import com.netflix.spinnaker.keel.PolicyRepository
+import com.netflix.spinnaker.keel.matcher.AllMatcher
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/v1/admin")
```

Comment:
What do you think about making this more descriptive? Like `/v1/policy`. Or, if you envision more admin things, changing the requests to be `/v1/admin/policy`

Reply:
Derp, yep - that's what I meant to do `/v1/admin/policy`. Will fix.